packages/project-access/src/file/findFiles.ts

```diff
@@ -39,8 +78,27 @@ export async function findFiles(filename: string, root: string, stopFolders: str
  *
  * @param fileName - file name to look for
  * @param startPath - path for start searching up
+ * @param fs - optional mem-fs-editor instance
+ * @returns - path to file name if found, otherwise undefined
+ */
+export function findFileUp(fileName: string, startPath: string, fs?: Editor): Promise<string | undefined> {
+    return Promise.resolve(findUp(fileName, startPath, fs ?? { exists }));
+}
+
+/**
+ * Internal find a file by name function for recursive iteration.
+ *
+ * @param fileName - file name to look for
+ * @param pathName - path for start searching up
+ * @param fs - mem-fs-editor instance
+ * @param fs.exists - function to be used to check the file existence
  * @returns - path to file name if found, otherwise undefined
  */
-export async function findFileUp(fileName: string, startPath: string): Promise<string | undefined> {
-    return findUp(fileName, { cwd: startPath });
+function findUp(fileName: string, pathName: string, fs: { exists: Editor['exists'] }): string | undefined {
+    const filePath = join(pathName, fileName);
+    if (fs.exists(filePath)) {
```

Comment:
@tobiasqueck, is there a reason why we have this not as `async function` and use function 
```
fileExists(path: string, memFs?: Editor)
```
from `fileAccess.ts`? I see you've even prepared the function to use with mem-fs.

Reply:
I prepared it after doing that one and totally forgot to consolidate the code.
Here it is now https://github.com/SAP/open-ux-tools/pull/705/commits/f8db29e7fbcb419f0cfad59b5b3f28dcb7d7379f